app.go

```diff
@@ -0,0 +1,19 @@
+//go:build !no_ui
```

Comment:
I wonder if instead of adding this build tag, more or less the same couldn't be achieved by just adding a `mkdir -p app/build && touch app/build/index.html` command to the `go-build-noui`?

Reply:
Going with the tag has the advantage of being able to build `litd` purely with `go build` whereas the approach you suggested needs additional shell commands.